spec/models/pacbio/well_library_factory_spec.rb

```diff
@@ -43,6 +43,8 @@
       expect(factory).to_not be_valid
       expect(factory.errors.full_messages).to_not be_empty
     end
+
+    it 'produces an error if there are multiples requests in any of the libraries and they do not have tags'
```

Comment:
Think the body of this test is missing?

Reply:
This was deliberate. I need to add a feature to library factory and well library factory to check for empty tags when it is multiplexed and didn't know the best way to do it. As it was not required for first pacbio release I left it. I will try and add something to the test to indicate that ...